### DISCUSSION TITLE
QOL change proposal : Add in a third talent specialization slot option

### DISCUSSION THREAD

**Discussion by evgeniNivanov** [chromiecraft/chromiecraft#6511] [Suggestions]
Specifically I want that change so that more people are interested in PvP . 
If you are mainly into PvE and have class that can fill in more than 1 role /which most of the server is / you would not bother with having your main/off spec be optimized for PvP . At best you will change talents before queuing for PvP . This leads to the server having a small core of people who have a full PvP spec (that they use offspec in PvE) that queues with the few PvE people interested in PvP (that may or may not bother to swap talents/glyphs) in order to start BG/arenas . A match that heavily favors the PvP core and tend to discourage nonPvPs to bother with PvP content . (Resilience meta problems and pvp gear availability are for another topic)
A solution to that imbalance will be to add in a third spec slot . 

With Northrend runes currently costing 330g / 35k honor it is cost prohibitive to change runes into a better set for pvp/pve and then reverting them back if needed . Even if you have Inscription and can make your own glyphs the swap is too expensive to bother with . A third slot for talents/glyphs will help people have an extra set that they may experiment with and not feel like gimping their main builds.

I know that there is an argument that this proposal is not blizzlike and will be a drain on the programming resources of the developers, but I feel that removing obstacles to get into PvP and incentivizing more people to want to queue for PvP events will be in the best interest of the server . Also Blizzard is bad for QoL changes - for example in Season of Discovery the main QoL things that people wanted were dual spec and lower respec costs .

**Comment by Never6** [comment]
as a druid player yes please

**Comment by CptMarkka** [comment]
As a Shaman main, I concur. Usually running PvE Enha and PvE Resto specs, and both specs have at least one Northrend glyph. Switching to casually play a bit of PvP here and there is prohibitively expensive, so most of the time I don't do it unless I anticipate I will be playing PvP for an extended period of time.

**Comment by EliteCow09** [comment]
I'm unsure of the technical requirements to implement a third spec, but it would be a nice quality of life improvement for veteran players looking to further min/max in PVE as well as PVP enjoyers.

If this helps remove a barrier from people trying PVP, where some specific glyphs are required to play optimally, I would be really interested in this as well if it helps get people into battlegrounds and arenas.

Ultimately, if it gives players more flexibility to play how they like (PVE or PVP), I think it's a worthwhile improvement to consider.

**Comment by elthehablo** [comment]
This is not something we can easily add. It would have to be via some form of command instead of the normal talent window as a command. In the best case scenario this would look quite ugly.

A sort of alternative to this would be a command to reset talents instead of going to the trainer. However this does not help with the glyph situation.

I'm sorry, but I think adding something like this is not very feasible.
